docs/index.asciidoc

```diff
@@ -35,7 +35,7 @@ include::dashboard.asciidoc[]
 
 include::timelion.asciidoc[]
 
-include::console.asciidoc[]
+include::index-shared3.asciidoc[]
```

Comment:
The index-shared* is related to broader changes to the index files (for re-use purpose), which will make sense in another PR.